Connection/Connection.php

```diff
@@ -650,7 +656,14 @@ public function isRemoteAddressConsidered()
      */
     public function getRemoteAddress()
     {
-        return $this->_remoteAddress;
+        $stream   = $this->getStream();
+        $streamId = (int) $stream;
```

Comment:
Stream is a php resource, and the integer attributed to a php resource is unique within the same process execution. (even through a fork)